x/pdv/types/params.go

```diff
@@ -14,21 +14,37 @@ const (
 
 var (
 	DefaultSupervisors = make([]string, 0)
+	DefaultMinGasPrice = sdk.NewDecCoinFromDec("stake", sdk.MustNewDecFromStr("0.025"))
 )
 
 var (
-	ParamSupervisors = []byte("supervisorsparams")
-	ParamFixedGas    = []byte("fixedgasparams")
+	KeySupervisors = []byte("Supervisors")
```

Comment:
since this moment it looks not like pdv module , I believe it's maintenance module or smth like that

Reply:
Module renamed to operations